.github/workflows/build-wheels-win32.yaml

```
@@ -14,14 +20,251 @@ concurrency:
+          python-version: "3.10"
+
+      - name: Install Python dependencies
+        shell: bash
+        run: |
+          python3 -m pip install wheel twine==5.0.0 setuptools
+
+      - name: Install sccache
+        run: choco install sccache -y
+
+      - name: Cache sccache
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)